skia/RSkComponentRNSVGTSpan.cpp

```diff
@@ -24,48 +40,97 @@ RnsShell::LayerInvalidateMask  RSkComponentRNSVGTSpan::updateComponentProps(Shar
 
 void RSkComponentRNSVGTSpan::onRender(const SkSVGRenderContext& ctx) const {
   if(content_.size()) {
+
     ParagraphStyle paragraph_style;
+
     ParagraphBuilderImpl fillBuilder(paragraph_style, fontCollection_);
     ParagraphBuilderImpl strokeBuilder(paragraph_style, fontCollection_);
-    TextStyle textStyle=getTextStyle();
-    SkRect frame=getContentFrame();
+    TextStyle textStyle=getContentTextStyle();
+    SkPoint frame=getContentDrawCoOrdinates();
     std::unique_ptr<Paragraph> paragraph;
```

Comment:
why unique pointer, can't we use the shared pointer.

Reply:
THis paragraph is not been shared anywhere , so it is unique.